html/semantics/embedded-content/media-elements/error-codes/error.html

```diff
@@ -28,6 +28,7 @@
       assert_true(e.error instanceof MediaError);
       assert_equals(e.error.code, 4);
       assert_equals(e.error.code, e.error.MEDIA_ERR_SRC_NOT_SUPPORTED);
+      assert_equals(typeof e.error.message, 'string');
```

Comment:
Since this will fail initially, can you add 'error.message type' as the description (third argument)? That will make the failures more intelligible.